### DISCUSSION TITLE
How to calculate the  attribute coefficient for multiple attributes?

### DISCUSSION THREAD

**Discussion by tedysetiadi** [networkx/networkx#4777] [Q&A]
I have graph with G, how to get the attribute_assortativity_coefficient multiple attributes together, for instance
G = nx.Graph()
G.add_node(0, age=10,sex="M")
G.add_node(1, age=20,sex="F")
G.add_node(2, age=20,sex="M")
G.add_node(3, age=30,sex="M")
G.add_edges_from([(0, 1),(1,2), (2,3)])
print(nx.attribute_assortativity_coefficient(G,"age","sex"))
the result = 0?

**Comment by dschult** [comment]
The documentation for [attribute_assortivity_coefficient](https://networkx.org/documentation/stable/reference/algorithms/generated/networkx.algorithms.assortativity.attribute_assortativity_coefficient.html) give a formula for this coefficient and a citation to a paper by Newman. But the formula is for a single attribute. I think you would need a different definition of assortativity to measure it with respect to 2 attributes. Perhaps you just want to do each one separately.
